### PR TITLE
Feature #1677:  Taxon definitions Nice to have.

### PR DESCRIPTION
**Pull request by nafiz1001**
https://206.12.92.46/issues/1677

### CODE REVIEW

**Review by ckostiw — CHANGES_REQUESTED** [review]
* Did we decide not to display taxon names in italics?
* The table needs to be updated after the user creates a new taxon.
* The `taxons` state should be refreshed as well

File: frontend/src/components/taxons/TaxonsListContent.tsx [hunk 1]
```diff
@@ -1,52 +1,49 @@
-import React, { useEffect, useState } from 'react'
+import React, { useEffect, useMemo, useState } from 'react'
```
Comment (ckostiw) [inline frontend/src/components/taxons/TaxonsListContent.tsx:1]: There are a lot of unused imports that should be cleaned up.

**Review by ckostiw — CHANGES_REQUESTED** [review]

**Comment by nafiz1001** [comment]
Oops.

**Comment by nafiz1001** [comment]
I'm surprised `taxons` state is not refreshed
```typescript
        if (isAdding) {
            dispatch(
                add({ ...new_taxon })
            )
                .then(() => {
                    navigate('/taxons')
                })
                .then(() => { dispatch(list()) })
		.then(() => { dispatch(TaxonsTableActions.setStale(true)) })
                .catch(err => setFormErrors({ ...err.data }))
        } else {
            dispatch(
                update(taxon.id, { ...new_taxon })
            )
                .then(() => {
                    navigate('/taxons')
                })
                .then(() => { dispatch(list()) })
		.then(() => { dispatch(TaxonsTableActions.setStale(true)) })		
                .catch(err => setFormErrors({ ...err.data }))
        }
```

**Review by ckostiw — APPROVED** [review]

**Pull request merged**